internal/pkg/tekton/pipeline_run_builder.go

```diff
@@ -118,6 +119,16 @@ func NewPipelineRunBuilder(name, namespace string) *PipelineRunBuilder {
 													Type: corev1.SeccompProfileTypeRuntimeDefault,
 												},
 											},
+											ComputeResources: corev1.ResourceRequirements{
+												Limits: corev1.ResourceList{
+													"cpu":    resource.MustParse("150m"),
+													"memory": resource.MustParse("512Mi"),
```

Comment:
The limits and requests are mixed up, requests should be lower or equal to the limits

Reply:
Ops! I inverted them, apologies. I've addressed the comment. May you have another look? Thanks.